package.json

```diff
@@ -60,6 +60,7 @@
     "resolve-url-loader": "^3.1.1",
     "sass-loader": "^8.0.2",
     "terser-webpack-plugin": "^2.3.5",
+    "ts-loader": "^6.2.1",
```

Comment:
no need to add "fork-ts-checker-webpack-plugin" also ?

Reply:
That's weird it does not crash without it

Reply:
ok then, anyway, it will be added by another PR